rs-libc/build.rs

```diff
@@ -71,3 +71,9 @@ fn main() {
 
     b.warnings(false).compile(name);
 }
+
+fn sorted<A: Ord, I: Iterator<Item=A>>(iterator: I) -> Vec<A> {
+    let mut items: Vec<_> = iterator.collect();
+    items.sort();
+    items
```

Comment:
You can sort while inserting:
```suggestion
fn sorted<A: Ord, I: Iterator<Item=A>>(iterator: I) -> BTreeSet<A> {
    iterator.collect()
```